app/src/main/java/com/sdpteam/connectout/common/EntityID.java

```diff
@@ -0,0 +1,35 @@
+package com.sdpteam.connectout.common;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Objects;
+
+public class EntityID {
+
+    protected final String value;
+
+    protected EntityID(String value) {
+        this.value = requireNonNull(value);
+    }
```

Comment:
I would argue that using protected instead of private for the attribute might be non-useful as the value is only given in the constructor.
and inversely, the constructor should be public, especially that IDs behaves in the same way, and should be instantiated through this class only.
What do you think about this ?